### PR TITLE
173 json schema support

### PR DESCRIPTION
**Pull request by nickjg231**
## Description
In this PR, initial support was added for VA JSON schemas. A fundamental part of how forms work on VA.gov is the use of jsonschema. Serving as a sort of contract between what the frontend sends and what the backend receives, jsonschemas help define what payload properties are: string, boolean, number, etc.

A core piece of Formik is supplying initialValues to use within a form, mapping DOM elements to form values for use with Formik's render methods.

VA Forms System Core needs to support the continued use of jsonschema. This PR has added initial support to use jsonschema with its ability to consume a given schema, extract its properties, and pass those into Formik as one object of initial values.

## Original issue(s)
[department-of-veterans-affairs/va-forms-system-core#0000](https://app.zenhub.com/workspaces/forms-library---platform-spike-team-61b0ae1f2cd3c30014e8a5b0/issues/department-of-veterans-affairs/va-forms-system-core/173)

## Screenshots
image
image
image
image

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link to the original github issue has been provided
- [ ] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/utils/helpers.ts [hunk 1]
```diff
@@ -28,3 +29,126 @@ export const buildRelativePath = (...args: string[]): string => {
...
+
+  /**
+   * For the passed key/value pair, flatten the value as much as possible, then append
+   * the key and updated value to the objectToAppendTo parameter
+   *
+   * @param objectToAppendTo
+   * @param key
+   * @param initialValue
+   */
+  appendPropery = (
```
Comment (bjmccotter7192) [inline src/utils/helpers.ts:107]: appendProperty* typo in function name

**Review by bjmccotter7192 — COMMENTED** [review]
This looks good @nickjg231, well done on the tests covering the different cases. Just found a typo but otherwise 👍

**Review by micahchiang — APPROVED** [review]
Looks good, excellent work @nickjg231! Just fix the typo @bjmccotter7192 mentioned and then feel free to merge 👍

**Comment by mlauren** [comment]
Looks good. I think docs still need to be generated for this one.

**Review by mlauren — APPROVED** [review]

**Pull request merged**